tests/navigation.spec.ts

```diff
@@ -0,0 +1,45 @@
+import { expect, test } from "@playwright/test";
+
+test.describe("Sidebar Navigation", () => {
+  test.describe("Under 'Desktop' resolution", () => {
+    test.describe.configure({ mode: "serial" });
+
+    test.beforeEach(async ({ page }) => {
+      await page.setViewportSize({ width: 1025, height: 900 });
+      await page.goto("http://localhost:3000/dashboard");
+    });
+
+    test("links work as expected", async ({ page }) => {
+      await expect(
+        page.getByRole("navigation").getByText("Projects"),
+      ).toHaveAttribute("href", "/dashboard");
+
```

Comment:
🌵 I'm thinking maybe this would work?

```suggestion
      const navigationPanel = page.getByRole('navigation');

      await expect(navigationPanel.getByText('Projects')).toHaveAttribute(
        'href',
        '/dashboard',
      );
      
```
It kinda cuts down on repeated code and processes.

Reply:
Hah, I _suppose_ we could even get a little spicy and do a little

```
      const textLinks = [
        { text: 'Projects', link: '' },
        { text: 'Issues', link: '/issues' },
        { text: 'Alerts', link: '/alerts' },
        { text: 'Users', link: '/users' },
        { text: 'Settings', link: '/settings' },
      ];

      await Promise.all(
        textLinks.map(async textLink => {
          return expect(
            navigationPanel.getByText(textLink.text),
          ).toHaveAttribute('href', `/dashboard${textLink.link}`);
        }),
      );
```

...but that's neither nor there.